modules/custom-status/custom-status.php

```diff
@@ -134,10 +137,9 @@ public function install() {
 			[
 				'term' => __( 'Pending Review' ),
 				'args' => [
-					'slug'               => 'pending',
-					'description'        => __( 'Post needs to be reviewed by an editor.', 'vip-workflow' ),
-					'position'           => 5,
-					'is_review_required' => true,
```

Comment:
`is_review_required` has been removed, as this can be computed from the presence of `required_user_ids`, and later other conditions like roles.